packages/dataviews/src/test/dataviews.tsx

```diff
@@ -445,66 +416,65 @@ describe( 'DataViews component', () => {
 			expect( mediaClickItemCallback ).toHaveBeenCalledWith( data[ 0 ] );
 		} );
 
-		it( 'accepts click for single selection', async () => {
+		it( 'accepts checkbox click for selection', async () => {
 			render(
 				<DataViewWrapper
 					view={ {
 						...DEFAULT_VIEW,
+						type: 'grid',
```

Comment:
It seems both of these tests under `grid` where testing the default view with `table` layout, and they were actually identical to the ones above - so not needed.

The changes here look good now.

The addition of the `mediaField` made me  realize that we don't render checkboxes without one. That doesn't seem right and I'd expect it to have been discussed already..

Can you loop me in about this and maybe an existing plan? Is `mediaField` required? I just tried in `pages` in trunk and you can remove the `Featured Image` from the view options and then there are no checkboxes and no visual indication that we have items selected.

Reply:
Yeah I was surprised by that as well. I wonder if it's related to the DataViewsPicker work. Since it's an issue on trunk do you think we should explore separately?